synapse-module/default.nix

```diff
@@ -79,6 +81,15 @@ in
       '';
     };
 
+    socketDir = mkOption {
+      type = types.path;
+      default = "/run/matrix-synapse";
+      description = ''
+        The directory where matrix-synapse by default stores the sockets of
+        all listeners that bind to UNIX sockets.
+      '';
+    };
```

Comment:
What is the benefit of making that configurable, instead of relying just on `RuntimeDirectory=`?

Reply:
No good reason really. I suppose we could reference `config.systemd.services.matrix-synapse.serviceConfig.RuntimeDirectory` instead. I find it hard to believe that anyone would ever want to configure it, unless they want to collect all sockets in one place for nginx to access or something.